rest/src/main/groovy/whelk/rest/api/CrudGetRequest.groovy

```diff
@@ -108,7 +108,8 @@ class CrudGetRequest {
     enum View {
         RESOURCE(''),
         DATA('data'),
-        DATA_VIEW('data-view');
+        DATA_VIEW('data-view'),
+        CHANGE_SETS('changesets');
```

Comment:
Since this is supposed to be an "internal" API (until further notice), we may want to call this `_changesets` (just to informally signal "don't depend on this")?

Reply:
:+1: fixed!